src/plugins/intel_gpu/include/intel_gpu/graph/kernel_impl_params.hpp

```diff
@@ -162,6 +163,32 @@ struct kernel_impl_params final {
         return _runtime_skippable;
     }
 
+    bool is_single_batch() const {
```

Comment:
Why do we need this function in kernel_impl_params? it is specific for fc. Can't we just move this to fully_connected_inst.h as a static function?

Reply:
I still cannot understand why we need to have this here. Why not just add this to fully_connected_inst.h as a static function such as 
bool is_single_batch(const kernel_impl_param& impl_param) so that we can call it anywhere?
Also seems that this function is only used to decide whether to use gemv or not. 
Then why not just fully_connected::use_gemv(const kernel_impl_param& impl_param) ? 
It would be preferable not to have a primitive specific handles in the kernel_impl_param

Reply:
Make sense, let me try to put it into fully_connected_inst.h.

Reply:
done